Add YouTube video picker (#34)
Kentico/custom-element-samples: YouTube/README.md

> # YouTube video selector

Reviewer: New line to follow https://github.com/DavidAnson/markdownlint/blob/v0.13.0/doc/Rules.md#md022
```suggestion
# YouTube video selector

```

YouTube video selector allows users to search and select videos from YouTube.

![YouTube video selector](YouTubeVideoSelector.gif)

> # Usage

Reviewer: Second level of heading to follow https://github.com/DavidAnson/markdownlint/blob/v0.13.0/doc/Rules.md#md025
```suggestion
## Usage
```

To use the YouTube video selector in your Kentico Cloud project:

* In Kentico Cloud open Content types tab
* Open / create a content model to which you want to add the YouTube video selector
* Add **Custom element** content element
* Open configuration of the content element
* Use following URL as Hosted code URL (HTTPS): https://kentico.github.io/custom-element-samples/YouTube/index.html

> * Provide the following JSON parameters for the custom element, to get an API key for the YouTube API vist https://developers.google.com/youtube/v3/getting-started

Reviewer: ```suggestion
* Provide the following JSON parameters for the custom element,
 to get an API key for the YouTube API visit https://developers.google.com/youtube/v3/getting-started
```

> ```

Reviewer: add code definition for code block

```suggestion
```json
```

{
  "apiKey": "<YOUR API KEY>"
}
```

> # Installation

Reviewer: Second level of heading to follow https://github.com/DavidAnson/markdownlint/blob/v0.13.0/doc/Rules.md#md025

```suggestion
## Installation
```

> YoutTube video selector source code is in following repository: https://github.com/Bassetts/kc-youtube

Reviewer: typo 
```suggestion
YouTube video selector source code is in following repository: https://github.com/Bassetts/kc-youtube
```

> If you want to customise the element please follow the instructions in source code repository README.md 

Reviewer: Typo
```suggestion
If you want to customize the element please follow the instructions in [source code repository `README.md`](https://github.com/Bassetts/kc-youtube/Readme.mhttps://github.com/Bassetts/kc-youtube/blob/master/README.md).
```

# YouTube video selector

YouTube video selector allows users to search and select videos from YouTube.

![YouTube video selector](YouTubeVideoSelector.gif)

## Usage

To use the YouTube video selector in your Kentico Cloud project:

* In Kentico Cloud open Content types tab
* Open / create a content model to which you want to add the YouTube video selector
* Add **Custom element** content element
* Open configuration of the content element
* Use following URL as Hosted code URL (HTTPS): https://kentico.github.io/custom-element-samples/YouTube/index.html
* Provide the following JSON parameters for the custom element,
 to get an API key for the YouTube API visit https://developers.google.com/youtube/v3/getting-started

```json
{
  "apiKey": "<YOUR API KEY>"
}
```

## Installation

YouTube video selector source code is in following repository: https://github.com/Bassetts/kc-youtube

> If you want to customize the element please follow the instructions in [source code repository `README.md`](https://github.com/Bassetts/kc-youtube/Readme.mhttps://github.com/Bassetts/kc-youtube/blob/master/README.md).

Reviewer: ```suggestion
If you want to customize the element please follow the instructions in [source code repository `README.md`](https://github.com/Bassetts/kc-youtube/Readme.md).
```
